independent-publisher-connectors/Strava/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2060 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Strava",
+    "description": "This is a connector for the Strava API",
```

Comment:
@RichardWie The description value seems vague and not more explanative about your service/product. Please provide a more detailed description that can be understandable by everyone and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)